src/EditorFeatures/Core.Wpf/Suggestions/FixAll/FixAllGetFixesService.cs

```diff
@@ -193,14 +193,15 @@ private static async Task<CodeAction> GetFixAllCodeActionAsync(IFixAllContext fi
 
 #endif
 
-                var changedSolution = previewService.PreviewChanges(
```

Comment:
I'm quite confused -- what ever guaranteed this was going to run on the UI thread before? Or was it always chance this whole time?

Reply:
AFAIK this was a chance the whole time.

Reply:
![image](https://user-images.githubusercontent.com/201340/195460476-0bfa3640-54e1-428f-b919-54605146d62d.png)